Bump version to v0.7.2 (#116)
Concordium/concordium-browser-wallet: packages/browser-wallet/CHANGELOG.md

### Fixed

> -   Recovered accounts not showing in reocvery results, if the identity was already present.

Reviewer: ```suggestion
-   Recovered accounts not showing in recovery results when the identity is already present.```
